utils/release/git-publish-all.mjs

```diff
@@ -73,12 +73,8 @@ process.chdir(process.env.INIT_CWD);
   // Current release branch
   // TODO v3: Bump to release/v3
   const currentReleaseBranch = 'release/v2';
-  await gitCheckoutBranch('release/v2');
-  await gitSetRefOnCommit(
-    'origin',
-    `refs/heads/${currentReleaseBranch}`,
-    commit
-  );
+  await gitCheckoutBranch(currentReleaseBranch);
+  await gitReset({resetMode: 'hard', ref: commit});
```

Comment:
This is the fix for why the release is red (and requires manual ops to complete)